lib/package/base/server-request/api/zuri_api.dart

```diff
@@ -102,10 +102,13 @@ class ZuriApi implements Api {
     }
   }
 
-  Future<dynamic> delete(String string) async {
+  Future<dynamic> delete(String string, {String? token}) async {
```

Comment:
Have you updated this function in the api.dart?
If not, please go and add what you added here in the api.dart file

Reply:
Done